src/components/LayoutDividerLine.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <hr
+    class="divider-line"
+    :class="[color, orientation]"
+    :orientation="orientation"
+    :color="color"
+  />
+</template>
+
+<script setup lang="ts">
+import {withDefaults, defineProps} from 'vue'
+
+export type Orientation = 'horizontal' | 'vertical'
+export type Color = 'gray' | 'light-gray'
+
+interface LayoutDividerLine {
+  orientation?: Orientation
+  color: Color
+}
+
+withDefaults(defineProps<LayoutDividerLine>(), {
+  orientation: 'horizontal',
+})
+</script>
+
+<style lang="scss">
+@import 'src/assets/styles/colors';
+
+.divider-line {
+  margin: 5px 0;
+
+  &.gray {
+    border: 1px solid $color-dusty-gray;
+  }
+
+  &.light-gray {
+    border: 1px solid $color-silver;
+  }
```

Comment:
Can we move the `1px solid` from both `&.gray` and `&.light-gray` to the basic `.divider-line`?
```suggestion
.divider-line {
  margin: 5px 0;
  border: 1px solid;

  &.gray {
    border-color: $color-dusty-gray;
  }

  &.light-gray {
    border-color: $color-silver;
  }
```